usecases/backup/backupper.go

```diff
@@ -103,7 +103,7 @@ func (b *backupper) OnStatus(ctx context.Context, req *StatusRequest) (reqStat,
 	meta, err := store.Meta(ctx, req.ID, false)
 	if err != nil {
 		path := fmt.Sprintf("%s/%s", req.ID, BackupFile)
-		return reqStat{}, fmt.Errorf("%w: %q: %v", errMetaNotFound, path, err)
+		return reqStat{}, fmt.Errorf("Backupper OnStatus: %w: %q: %v", errMetaNotFound, path, err)
```

Comment:
I would recommend against altering the error message. There's no need to incorporate debug information, such as the function name, in a message that is returned to public API users.

The principle thing applies to the changes below.

Reply:
There is most definitely a need for altering the error message.  There are five separate places where this error message appears, and the error messages are completely indistinguishable.  Since weaviate strips all location information from the errors, there is no way to tell them apart.

Good, informative error messages are vital to good quality engineering, so there is a need to incorporate this information, and so these changes are required.

Reply:
> There is most definitely a need for altering the error message. There are five separate places where this error message appears, and the error messages are completely indistinguishable. Since weaviate strips all location information from the errors, there is no way to tell them apart.

First, it's worth mentioning that the current message is intended for the public API.

> Good, informative error messages are vital to good quality engineering, so there is a need to incorporate this information, and so these changes are required.

However, while I certainly value the importance of informative error messages, it seems that the new message might not be the most suitable for a couple of reasons:

1. It appears to leak internal debugging information to public API users.
2. Its format resembles that of a logging message rather than one intended for the public AP

Having said that, I am not against changing the message to a more descriptive one if it can provide meaningful information to users without exposing internal details.

Reply:
Huh, replying to emails doesn't reply here.

I agree it would be nice to have simple messages for the public API, but is there a way we could also have good debugging messages for debugging?

Also
> It appears to leak internal debugging information to public API users.

we're open source, we don't have to worry about that